docs/source/apidoc/kliff.rst

```diff
@@ -4,16 +4,3 @@ Package Reference
 =================
 
 .. toctree::
```

Comment:
If these are removed, will the docs for API be excluded?

Would be great to check it locally.